python/ray/tune/integration/xgboost.py

```diff
@@ -3,6 +3,20 @@
 
 import os
 
+try:
+    from xgboost_ray.session import is_xgboost_ray_actor, get_actor_rank, \
+        put_queue
+except ImportError:
+
```

Comment:
Can you add some comments here about what you are doing (just for external users to read)